src/pages/PoolsPage/PoolTable.tsx

```diff
@@ -0,0 +1,163 @@
+import Table from "../../components/organism/Table";
+import Button from "../../components/atom/Button";
+import { ReactComponent as AddToken } from "../../assets/img/addIcon.svg";
+import { ReactComponent as DotToken } from "../../assets/img/dotToken.svg";
+import { ReactComponent as TestToken } from "../../assets/img/testToken.svg";
+import { ButtonText, ButtonVariants } from "../../global/enum";
+
+const PoolMock = [
+  {
+    name: "DOT–DAI",
+    tlv: "$1.26m",
+    tokenAIcon: <DotToken width={36} height={36} />,
+    tokenBIcon: <TestToken width={36} height={36} />,
+    totalTokensLocked: {
+      tokenA: "234.42",
+      tokenB: "436.42",
+      tokenAIcon: <DotToken width={16} height={16} />,
+      tokenBIcon: <TestToken width={16} height={16} />,
+    },
+    deposited: {
+      tokenA: "234.42",
+      tokenB: "436.42",
+      tokenAIcon: <DotToken width={16} height={16} />,
+      tokenBIcon: <TestToken width={16} height={16} />,
+    },
+  },
+  {
+    name: "DOT–BUSD",
+    tlv: "$15.05m",
+    tokenAIcon: <DotToken width={36} height={36} />,
+    tokenBIcon: <TestToken width={36} height={36} />,
+    totalTokensLocked: {
+      tokenA: "234.42",
+      tokenB: "436.42",
+      tokenAIcon: <DotToken width={16} height={16} />,
+      tokenBIcon: <TestToken width={16} height={16} />,
+    },
+  },
+  {
+    name: "DOT–ATID",
+    tlv: "$103.21m",
+    tokenAIcon: <DotToken width={36} height={36} />,
+    tokenBIcon: <TestToken width={36} height={36} />,
+    totalTokensLocked: {
+      tokenA: "234.42",
+      tokenB: "436.42",
+      tokenAIcon: <DotToken width={16} height={16} />,
+      tokenBIcon: <TestToken width={16} height={16} />,
+    },
+  },
+  {
+    name: "DOT–WETH",
+    tlv: "$24.35m",
+    tokenAIcon: <DotToken width={36} height={36} />,
+    tokenBIcon: <TestToken width={36} height={36} />,
+    totalTokensLocked: {
+      tokenA: "234.42",
+      tokenB: "436.42",
+      tokenAIcon: <DotToken width={16} height={16} />,
+      tokenBIcon: <TestToken width={16} height={16} />,
+    },
+  },
+  {
+    name: "DOT–ASTR",
+    tlv: "$193.45m",
+    tokenAIcon: <DotToken width={36} height={36} />,
+    tokenBIcon: <TestToken width={36} height={36} />,
+    totalTokensLocked: {
+      tokenA: "234.42",
+      tokenB: "436.42",
+      tokenAIcon: <DotToken width={16} height={16} />,
+      tokenBIcon: <TestToken width={16} height={16} />,
+    },
+  },
+  {
+    name: "DOT–USDT",
+    tlv: "$265.35m",
+    tokenAIcon: <DotToken width={36} height={36} />,
+    tokenBIcon: <TestToken width={36} height={36} />,
+    totalTokensLocked: {
+      tokenA: "234.42",
+      tokenB: "436.42",
+      tokenAIcon: <DotToken width={16} height={16} />,
+      tokenBIcon: <TestToken width={16} height={16} />,
+    },
+  },
+];
```

Comment:
Could u wrap this with Promise? We need that cuz of the load state.
smth like this
const fetchData = () => {
  return new Promise((resolve) => {
    setTimeout(() => {
      const mockData = [
        { id: 1, title: 'Mock Item 1' },
        { id: 2, title: 'Mock Item 2' },
        { id: 3, title: 'Mock Item 3' },
      ];
      resolve(mockData);
    }, 1000); // Simulating a delay of 1 second
  });
};

Reply:
done